erc-64.md

```diff
@@ -0,0 +1,153 @@
+---
+eip: xx
```

Comment:
```suggestion
eip: 7734
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
done